api-report/request-handler.api.md

```diff
@@ -17,14 +18,14 @@ import { RequestParser } from '@fluidframework/runtime-utils';
 export function buildRuntimeRequestHandler(...handlers: RuntimeRequestHandler[]): (request: IRequest, runtime: IContainerRuntime) => Promise<IResponse>;
 
 // @public (undocumented)
-export const createFluidObjectResponse: (fluidObject: IFluidObject) => {
+export const createFluidObjectResponse: (fluidObject: FluidObject) => {
     status: number;
     mimeType: string;
-    value: IFluidObject;
+    value: Partial<Pick<unknown, never>>;
```

Comment:
Maybe this function needs explicit typing to avoid the weird return type in documentation?